pkgs/build-support/rust/hooks/cargo-setup-hook.sh

```diff
@@ -61,14 +61,15 @@ cargoSetupPostPatchHook() {
       fi
 
       echo
-      echo "ERROR: cargoSha256 is out of date"
+      echo "ERROR: cargoHash or cargoSha256 is out of date"
```

Comment:
I'd argue that this change could be in the same commit, assuming you mention it.